client/src/components/OverviewComponent/style.css

```diff
@@ -0,0 +1,32 @@
+.overview-container {
+    display: flex;
+    justify-content: space-around;
+    width: 100vw;
+    height: 100vh;
+}
+
+.container-photo {
+    width: 50%;
+}
+
+.container-text {
```

Comment:
Should this be `.container-content`? I don't see the className `container-text` in the component anywhere.

Reply:
yes its .container-content .. i forget to change it :sweat_smile: